website/docs/r/organizations_policy.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "aws"
+page_title: "AWS: aws_organizations_policy"
+sidebar_current: "docs-aws-resource-organizations-policy"
+description: |-
+  Provides a resource to manage an AWS Organizations policy.
+---
+
+# aws_organizations_policy
+
+Provides a resource to manage an [AWS Organizations policy](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies.html).
+
+## Example Usage
+
+```hcl
+data "aws_iam_policy_document" "example" {
+  statement {
+    actions   = ["*"]
+    resources = ["*"]
+  }
+}
+
+resource "aws_organizations_policy" "example" {
+  content = "${data.aws_iam_policy_document.example.json}"
```

Comment:
I wonder if it will be confusing to be including an IAM policy as a Service Control Policy, even though they are compatible...?

Reply:
I think its better to show the "easier" way to manage policies rather than error-prone heredoc syntax personally.